integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -25,6 +25,10 @@ user: 1000
 group: 1000
 fsGroup: 1000
 
+# Mount path in the host
+#mountPath: /mnt/alluxio-fuse
+mountPath: /mnt
```

Comment:
please include this path under the `fuse` section
```
fuse:
  mountPath: /mnt
```

Reply:
Sure.